app/templates/settings/integration.hbs

```diff
@@ -192,16 +192,15 @@
                 <tr class="bt b--whitegrey" data-test-webhooks-blank-slate>
                     <td colspan="5" class="pa5 pt15 pb15 tc midgrey f7">
                         <span class="db">No webhooks configured</span>
-                        <span class="dib mt1">
+                        {{!-- <span class="dib mt1">
```

Comment:
Is it useful to leave this commented out or can we remove it completely?

Reply:
sorry, missed this when I cleaned up my code. I didn't intend to keep it there...